bazel/envoy_build_system.bzl

```diff
@@ -455,15 +455,17 @@ def envoy_cc_test(
     test_lib_tags = []
     if coverage:
         test_lib_tags.append("coverage_test_lib")
-    envoy_cc_test_library(
-        name = name + "_lib",
+    _envoy_cc_test_infrastructure_library(
+        name = name + "_lib_INTERNAL_ONLY",
```

Comment:
I think `_internal_only` is more consistent with our usual naming and still clear enough.

Reply:
Done.